x-pack/plugins/dashboard_enhanced/kibana.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "dashboardEnhanced",
+  "version": "kibana",
+  "server": true,
+  "ui": true,
+  "requiredPlugins": ["uiActions", "embeddable", "drilldowns"],
```

Comment:
add `dashboard` on here?  Then if dashboard is disabled, so is this.

Reply:
Done.